src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">
+        {comment.name}
```

Comment:
Consider using destructuring assignment for `comment` in this component.

Read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)